database/seeds/UserTableSeeder.php

```diff
@@ -11,6 +13,25 @@ class UserTableSeeder extends Seeder
      */
     public function run()
     {
-        factory(Gladiator\Models\User::class, 5)->create();
+        // Add Admin Users
+        for ($i = 0; $i < 2; $i++) {
+            User::create([
+                'id' => str_random(24),
+                'role' => 'admin',
+            ]);
+        }
+
+
+        // Add Contestant Users
+        $waitingRooms = WaitingRoom::all();
+        $totalRooms = count($waitingRooms);
```

Comment:
I generally use :snake: when referring to a database column or URL parameter & :camel: for everything else.